packages/location-field/src/index.js

```diff
@@ -308,17 +318,71 @@ class LocationField extends Component {
       });
   }
 
-  render() {
+  renderFeature = (itemIndex, feature) => {
     const {
+      layerColorMap,
+      geocoderConfig,
       addLocationSearch,
+      GeocodedOptionIconComponent,
+      operatorIconMap
```

Comment:
Sort vars

Reply:
Sorted in e0fc8a811358ed8a55a587f948db1614da68dbd4